recipe/meta.yaml

```diff
@@ -31,24 +30,25 @@ build:
 requirements:
     build:
         - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
         - {{ compiler('dpcpp') }}
         - patch     # [not win]
         - m2-patch  # [win]
         - cmake >=3.21
-        - git
+        - git  # [not win]
         - ninja
         - cython
     host:
         - setuptools
         - cython
-        - pybind11 2.9.2
+        - pybind11 2.10.4
         - python
         - scikit-build
-        - numpy 1.21
+        - numpy 1.22
```

Comment:
Does the upstream specify explicitly numpy 1.22 (can't see it)? Could use whatever is defined in aggregate cbc.

Reply:
These packages are not on conda-forge. They are in the intel channel. The upstream is https://github.com/IntelPython/dpctl/blob/master/conda-recipe/conda_build_config.yaml#L2. Intel also haven't released dpctl on PyPI for a while, see https://pypi.org/project/dpctl/#history.

Reply:
ah sorry, I saw 'conda' and did not read the rest.